Added release notes

gaurav-nelson opened the pull request:
Based on https://help.stackrox.com/docs/release-notes/

Preview: 
- https://openshift-docs-git-release-notes-gnelson.vercel.app/openshift-acs/master/release_notes/3061-release-notes.html
- https://openshift-docs-git-release-notes-gnelson.vercel.app/openshift-acs/master/release_notes/3060-release-notes.html
- https://openshift-docs-git-release-notes-gnelson.vercel.app/openshift-acs/master/release_notes/3059-release-notes.html
- https://openshift-docs-git-release-notes-gnelson.vercel.app/openshift-acs/master/release_notes/3058-release-notes.html
- https://openshift-docs-git-release-notes-gnelson.vercel.app/openshift-acs/master/release_notes/3057-release-notes.html
- https://openshift-docs-git-release-notes-gnelson.vercel.app/openshift-acs/master/release_notes/3056-release-notes.html

gaurav-nelson commented:
@bergerhoffer PTAL

Please note that I did not modify them a lot. We want to keep release notes for the last 6 releases.

bergerhoffer commented on _topic_map.yml:
```diff
@@ -29,6 +29,23 @@ Topics:
 - Name: Welcome
   File: index
 ---
+Name: Release Notes
```
Release notes

bergerhoffer commented on release_notes/3056-release-notes.adoc:
```diff
@@ -0,0 +1,124 @@
+
+*Release date:* February 24, 2021
+
+== New features
```
I think we're supposed to add explicit anchors for all of these headings, both in modules and assemblies

bergerhoffer commented on release_notes/3056-release-notes.adoc:
```diff
@@ -0,0 +1,124 @@
+
+=== Network baseline updates
+
+The {product-title} discovers your deployments' network flows and creates a baseline based on the regular network activity in your clusters.
```
Should "The" be removed at the beginning of the sentence?

bergerhoffer commented on release_notes/3056-release-notes.adoc:
```diff
@@ -0,0 +1,124 @@
+=== Network baseline updates
+
+The {product-title} discovers your deployments' network flows and creates a baseline based on the regular network activity in your clusters.
+You can now configure alerts and block network activity for connections that don't exist in the baseline.
```
don't -> do not

bergerhoffer commented on release_notes/3056-release-notes.adoc:
```diff
@@ -0,0 +1,124 @@
+
+== Important bug fixes
+
+* *ROX-3893*: Previously, if you used `--json` option with the `roxctl image check` command, it would always exit with code `0`.
```
used -> used the

bergerhoffer commented on release_notes/3056-release-notes.adoc:
```diff
@@ -0,0 +1,124 @@
+
+* *ROX-6504*: Previously, the RHACS portal would forcefully log out users when they selected *Platform Configuration* -> *Access Control*, and they did not have appropriate permissions.
+
+* *ROX-6522*: Fixed an issue in the RHACS portal, where the component's location would not display if you were viewing details for a specific image.
```
Comma could be removed. Check other where sentences too

bergerhoffer commented on release_notes/3056-release-notes.adoc:
```diff
@@ -0,0 +1,124 @@
+
+* *ROX-6699*: Previously, modifying the *Docker CIS 5.15: Ensure that the host's process namespace is not shared* policy would result in an error.
+
+* *ROX-6716*: Previously, if you have used Helm to install a Sensor, you could not enable automatic upgrades for that secured cluster.
```
you have used -> you used

bergerhoffer commented on release_notes/3056-release-notes.adoc:
```diff
@@ -0,0 +1,124 @@
+* *ROX-6716*: Previously, if you have used Helm to install a Sensor, you could not enable automatic upgrades for that secured cluster.
+You can now use the `SENSOR_HELM_NOT_HELM_MANAGED=true` environment variable to enable automatic upgrades for Helm managed secured clusters.
+
+== Known Issues
```
Issues -> issues

bergerhoffer commented on release_notes/3056-release-notes.adoc:
Should this link be removed or something different?

bergerhoffer commented on release_notes/3056-release-notes.adoc:
`>` -> `->`

bergerhoffer commented on release_notes/3056-release-notes.adoc:
colon instead of the comma

bergerhoffer commented on release_notes/3057-release-notes.adoc:
```diff
@@ -0,0 +1,101 @@
+* *ROX-6085*: Previously, setting Central log level by using the `roxctl` CLI's `debug log` command would not work sometimes.
+You could also set unacceptable values for log level (for example, `Trace`), which did not affect the log level.
+
+* *ROX-6302*: The *Violations* view where loading too many violations would sometimes crash the RHACS portal page.
```
Just checking, is "where" the right word? I'm having trouble understanding what the sentence is trying to say. Should it be "Loading too many violations on the **Violations** view would sometimes..."

bergerhoffer commented on release_notes/3057-release-notes.adoc:
```diff
@@ -0,0 +1,101 @@
+{product-title} now includes dry run support for the admission controller webhook.
+
+* *ROX-6640*: The RHACS portal would not display the full description for RHSA CVEs in the *CVE* details view.
+* *ROX-6723*: Previously, if you were using the default CA certificate for Central, and you have configured an additional CA certificate for a Sensor, the {product-title} would overwrite Sensor's additional certificate.
```
you have -> you

bergerhoffer commented on release_notes/3057-release-notes.adoc:
```diff
@@ -0,0 +1,101 @@
+* *ROX-6640*: The RHACS portal would not display the full description for RHSA CVEs in the *CVE* details view.
+* *ROX-6723*: Previously, if you were using the default CA certificate for Central, and you have configured an additional CA certificate for a Sensor, the {product-title} would overwrite Sensor's additional certificate.
+
+* *ROX-6736*: Previously, sometimes the RHACS portal didn't show allowed connections between Sensor and non-isolated deployments in the *Network Graph*  view.
```
didn't -> did not

bergerhoffer commented on release_notes/3057-release-notes.adoc:
```diff
@@ -0,0 +1,101 @@
+* *ROX-6834*:  Previously, in {product-title} version 3.0.57.0, you could not install a Sensor using the RHACS portal because downloading the Sensor bundle would fail.
+
+* *ROX-6805*: Previously, if you were using {product-title} on {ocp}, the security context constraint link:https://bugzilla.redhat.com/show_bug.cgi?id=1824800[conflicted with the Authentication Operator].
+{product-title} now includes updated security context constraint which fixes this issue.
```
now includes -> now includes an

bergerhoffer commented on release_notes/3057-release-notes.adoc:
A random parenthesis here

bergerhoffer commented on release_notes/3058-release-notes.adoc:
```diff
@@ -0,0 +1,73 @@
+* *ROX-6626*: Previously, if you were using {product-title} on {ocp}, the *Network Graph* view would show too many connections and did not show Network baselines.
+
+* *ROX-6792*: Fixed an issue with the *inactive deployment* filter in the *Violations* view.
+* *ROX-6820*: Previously, {product-title} would not report CVE's in Distroless images under certain conditions.
```
CVS's -> CVEs

bergerhoffer commented on release_notes/3058-release-notes.adoc:
```diff
@@ -0,0 +1,73 @@
+== Important bug fixes
+
+* *ROX-5397*, *ROX-6458*, and *ROX-6619*: Fixed a minor issue in the *Iptables Executed in Privileged Container* security policy, updated the remediation instructions in the *Curl in Image* security policy, and updated the *Kubernetes Dashboard Deployed* policy criteria.
+* *ROX-6497*: Previously, you could not use  OIDC Identity Provider with the Authorization Code Grant authentication flow, by using a client secret.
```
should Identity Provider be lowercase? and maybe an "an" after "use"?

bergerhoffer commented on release_notes/3058-release-notes.adoc:
```diff
@@ -0,0 +1,73 @@
+Release date: Apr 20, 2021
+
+* *ROX-6959*: Previously, the {ocp} Cluster Version Operator was not correctly identified as an orchestrator component.
+We've fixed this issue.
```
Rewrite all of the "we've" to get rid of "we"

bergerhoffer commented on release_notes/3058-release-notes.adoc:
```diff
@@ -0,0 +1,73 @@
+
+* *ROX-6959*: Previously, the {ocp} Cluster Version Operator was not correctly identified as an orchestrator component.
+We've fixed this issue.
++
```
This + should be removed; the security updates heading isn't rendering properly

bergerhoffer commented on release_notes/3058-release-notes.adoc:
```diff
@@ -0,0 +1,73 @@
++
+== Security updates
+
+We've updated the Collector image to resolve the following fixable CVEs:
```
here and next sentence, should it be singular CVE/RHSA

bergerhoffer commented on release_notes/3058-release-notes.adoc:
`>` -> `->` for all of these?

bergerhoffer commented on release_notes/3059-release-notes.adoc:
```diff
@@ -0,0 +1,64 @@
+== Important bug fixes
+
+* *ROX-6696*: {product-title} includes updates to AWS Security Hub integration to respect Amazon's rate limits on payload size to prevent errors.
+* *ROX-5299*, *ROX-6496*, *ROX-6856*, and *ROX-6718* *ROX-6472*:{product-title} includes updated Sensor which fixes multiple data race conditions, which resulted in errors.
```
"and" is in the wrong spot

bergerhoffer commented on release_notes/3059-release-notes.adoc:
```diff
@@ -0,0 +1,64 @@
+
+Release date: May 4, 2021
+
+* *ROX-7053*: Previously, re-using a cluster name might cause autogenerated registry integrations to get duplicated.
```
reusing

bergerhoffer commented on release_notes/3059-release-notes.adoc:
```diff
@@ -0,0 +1,64 @@
+
+Release date: May 13, 2021
+
+* *ROX-7154*: Previously, when using standalone pods with attached services customers would sometimes experience unstable deployments and crash loops.
```
Comma after services?

bergerhoffer commented on release_notes/3059-release-notes.adoc:
```diff
@@ -0,0 +1,64 @@
+== Important system changes
+
+* *ROX-7016*: Product is now rebranded as {product-title}.
+* *ROX-6909*: The *Curl in image* and *Wget in image* policies and now disabled by default to reduce default violations.
```
and now -> are now ?

bergerhoffer commented on release_notes/3060-release-notes.adoc:
```diff
@@ -0,0 +1,62 @@
+
+*Release date:* May 19, 2021
+
+== New Features
```
Features -> features (a few other files too)

bergerhoffer commented on release_notes/3060-release-notes.adoc:
```diff
@@ -0,0 +1,62 @@
+
+== Important system changes
+
+* *ROX-7059*: {product-title} includes updated user interface to show resource summary counts after removing them in 3.0.58.0.
```
includes -> includes an

bergerhoffer commented on release_notes/3061-release-notes.adoc:
prematurely

bergerhoffer commented on release_notes/3061-release-notes.adoc:
Is "failing open" the right words here?

bergerhoffer commented on release_notes/3061-release-notes.adoc:
Just checking, does "Image" need to be capitalized?

gaurav-nelson commented:
Thank you @bergerhoffer for reviewing. I've addressed you comments.

Please note that in the latest updated:
- I've added a new file `release_notes/3062-release-notes.adoc`.
- removed `release_notes/3056-release-notes.adoc` from the TOC as we plan to keep only the latest 6 release notes.

bergerhoffer commented:
LGTM, thanks for the updates.
